src/types/dict.rs

```diff
@@ -557,76 +557,73 @@ pub(crate) use borrowed_iter::BorrowedDictIter;
 
 /// Conversion trait that allows a sequence of tuples to be converted into `PyDict`
 /// Primary use case for this trait is `call` and `call_method` methods as keywords argument.
-pub trait IntoPyDict: Sized {
+pub trait IntoPyDict<'py>: Sized {
```

Comment:
I guess the alternative is keep the existing form around for a while and add a new trait `IntoPyDictObject` or similar which we can eventually rename back to just this. But I'm unsure how much value there really is in that, for this small corner of the API.

Reply:
I have the feeling that this is mostly used to get access to the constructors we provide and is probably not often implemented on custom types. So I think it would likely cause more churn doing the renaming business.